osbuild/meta.py

```diff
@@ -641,8 +643,14 @@ def detect_runner(self, name, fallback="org.osbuild.linux") -> RunnerInfo:
             candidate = candidates.pop()
 
             if candidate.version == version:
-                return candidate.distro
-            elif candidate.version < version:
+                return candidate
```

Comment:
So this was previously wrong, therefore the line change should be really moved to the first commit.